### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by paulo-coutinho** [cross-language-cpp/djinni-generator#13]
Hi,

What is the project status?

What it have more than the oficial and abandoned project?

Im using this fork "https://github.com/hiennguyenle/finn" that you can get some fixes and changes.

Thanks.

**Comment by a4z** [comment]
please see https://github.com/cross-language-cpp/djinni/issues/22

**Comment by jothepro** [comment]
Improvements for the generator in this fork:
- a standalone executable: No more executing a dubios `run` script in some subfolder. No more building djinni yourself. Just download the binary from the [latest release](https://github.com/cross-language-cpp/djinni-generator/releases/latest) and use it: `djinni --help`. Works on Windows, macos and linux!
- Updated dependencies to more recent versions. I am planning to continue this kind of maintenance. E.g. to update the `scopt` library some refactoring is required.
- Updated project structure: The project structure & sbt build tool version have been updated. Everything is cleaner, more up-to-date and more standard now.
- Improved QA: The previous generator implementation itself had no tests. All that existed was some super-high level Integration testing. I added some tests that ensure the output of the generator stays consistent. Tests are automatically executed before in each PR in github.

As you see most changes have happened under the hood. The biggest advancements for the end-user is having a pre-built binary at hand. The roadmap is not to make super big changes but to give the code a facelift to make it future-proof.

But there is more to it! We have bigger plans in cross-language-cpp:
- Updated example projects for iOS/Android
- Improved/updated documentation
- Improved Build-Tool integration: https://github.com/cross-language-cpp/djinni/issues/10

I can't recommend you to switch to our fork yet, as the splitting of the original repo is still in progress. Especially the [support-lib](https://github.com/cross-language-cpp/djinni-support-lib) needs some more work. But I hope you stay tuned and give feedback!

Btw. if you want to be part of our organisation and contribute ideas or code to it I'd be super excited!!!
